packages/hunspell_fr_fr.rb

```diff
@@ -3,42 +3,30 @@
 class Hunspell_fr_fr < Package
   description 'Dictionnaire Français Hunspell'
   homepage 'http://hunspell.github.io/'
-  version '1.7.0-2'
+  version '25.8.3.2'
   license 'MPL-1.1, GPL-2 and LGPL-2.1'
   compatibility 'all'
-  source_url 'https://github.com/hunspell/hunspell/archive/v1.7.0.tar.gz'
-  source_sha256 'bb27b86eb910a8285407cf3ca33b62643a02798cf2eef468c0a74f6c3ee6bc8a'
-  binary_compression 'tar.xz'
+  source_url 'https://github.com/LibreOffice/dictionaries.git'
+  git_hashtag "libreoffice-#{version}"
+  binary_compression 'tar.zst'
 
   binary_sha256({
-    aarch64: 'f9436296b3b3cc53ed1307e21493754054037fa4ee69e8d047316d7d9906ce07',
-     armv7l: 'f9436296b3b3cc53ed1307e21493754054037fa4ee69e8d047316d7d9906ce07',
-       i686: '3d3d35a011256158f200ea5761ac3562f534733865a03c6f1023284e79235bca',
-     x86_64: 'aea9ec478d565bb9b240157630ae265e11f108652293131b439c8bf43ed546b9'
+    aarch64: '4668fb78a8d93315d7dde45616d7e1ee885ffa8a3105cdf26502ed721f260c49',
+     armv7l: '4668fb78a8d93315d7dde45616d7e1ee885ffa8a3105cdf26502ed721f260c49',
+       i686: 'c70e2022033ece666397460e651198b0ba116afbcc1e7c5d23eedbdb1299b101',
+     x86_64: 'dbd5dfce530cbcb9ebd4da43691d7d70f3eb95c0e2f51912478be7179a86e76c'
   })
 
   depends_on 'hunspell_base'
 
   def self.install
-    system 'git clone -b libreoffice-6-1 --depth 1 git://anongit.freedesktop.org/libreoffice/dictionaries.git'
-    system "install -Dm644 dictionaries/fr_FR/fr.aff #{CREW_DEST_HOME}/Library/Spelling/fr_FR.aff"
-    system "install -Dm644 dictionaries/fr_FR/fr.dic #{CREW_DEST_HOME}/Library/Spelling/fr_FR.dic"
+    FileUtils.install 'fr_FR/fr.aff', "#{CREW_DEST_HOME}/Library/Spelling/fr_FR.aff", mode: 0o644
+    FileUtils.install 'fr_FR/fr.dic', "#{CREW_DEST_HOME}/Library/Spelling/fr_FR.dic", mode: 0o644
 
     FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/env.d/"
-    @env = <<~EOF
+    File.write "#{CREW_DEST_PREFIX}/etc/env.d/hunspell_fr_fr", <<~EOF
       # Configuración de hunspell
       export DICTIONARY=fr_FR
     EOF
-    File.write("#{CREW_DEST_PREFIX}/etc/env.d/hunspell_fr_fr", @env)
-  end
-
-  def self.postinstall
-    system 'git clone -b libreoffice-6-1 --depth 1 git://anongit.freedesktop.org/libreoffice/dictionaries.git'
-    FileUtils.cp 'dictionaries/fr_FR/fr.aff', "#{HOME}/Library/Spelling/fr_FR.aff"
-    FileUtils.cp 'dictionaries/fr_FR/fr.dic', "#{HOME}/Library/Spelling/fr_FR.dic"
-    FileUtils.rm_rf 'dictionaries/'
-    puts
-    puts "To update the dictionaries periodically, execute 'crew postinstall hunspell_fr_fr'".lightblue
-    puts
   end
```

Comment:
Should there be a postremove section to remove the database?

Reply:
Why? The files are in the filelist.